chartmogul/chartmogul-node
# Fix bug in assertions returning true for some tests

**Previous issues**
Most of the existing tests were written incorrectly. 
The `assertions` are inside a promise which does not return. The `test` will always return true.

**Improvements Done**
1. All `tests` which does not return anything were rewritten
2. `Await/async` used instead of `done/done()` for deprecation error tests


**How to properly test**
To properly test;
1. Clone the repo
2. Change any assertion
3. Run the test
4. The test should fail e.g



For more context, here is the shortcut ticket
https://app.shortcut.com/chartmogul/story/57880/rewrite-the-assertion-in-chartmogul-node-library
